TomTomSDKExamples/Examples/UseCase/BasicNavigationMenu.swift

```diff
@@ -0,0 +1,172 @@
+//  license agreement between you and TomTom NV. If you are the licensee, you are only permitted
+//  to use this software in accordance with the terms of your license agreement. If you are
+//  not the licensee, you are not authorized to use this software in any manner and should
+//  immediately return or destroy it.
+
+import SwiftUI
+
+struct BasicNavigationMenu: View {
+    var body: some View {
+        ZStack(alignment: .topLeading, content: {
+            VStack(alignment: .center, spacing: 16) {
+                Text(Resource.Strings.options)
+                    .foregroundColor(Resource.Colors.titleLight)
+                    .font(.title2)
+                    .fontWeight(.semibold)
+                    .padding(.vertical, 20)
+
+                HStack(alignment: .center, spacing: 16) {
+                    Rectangle().frame(height: 1)
+                        .foregroundColor(Resource.Colors.inputFieldLight)
+
+                    Text(Resource.Strings.mapMode)
+                        .fontWeight(.semibold)
+                        .foregroundColor(Resource.Colors.bodyLight)
+
+                    Rectangle().frame(height: 1)
+                        .foregroundColor(Resource.Colors.inputFieldLight)
+                }
+
+                HStack {
+                    // Online
+                    VStack(alignment: .center, spacing: 16) {
+                        Image(Resource.Images.online)
+                            .modifier(MenuModifier(isSelected: selectedMapMode == .online))
+
+                        Text(Resource.Strings.online)
+                            .foregroundColor(Resource.Colors.bodyLight)
+                    }
+                    .onTapGesture {
+                        selectedMapMode = .online
+                    }
+
+                    // Hybrid
+                    VStack(alignment: .center, spacing: 16) {
+                        Image(Resource.Images.hybrid)
+                            .modifier(MenuModifier(isSelected: selectedMapMode == .hybrid))
+
+                        Text(Resource.Strings.hybrid)
+                            .foregroundColor(Resource.Colors.bodyLight)
+                    }
+                    .onTapGesture {
+                        selectedMapMode = .hybrid
+                    }
+
+                    // Offline
+                    VStack(alignment: .center, spacing: 16) {
+                        Image(Resource.Images.offline)
+                            .modifier(MenuModifier(isSelected: selectedMapMode == .offline))
+
+                        Text(Resource.Strings.offline)
+                            .foregroundColor(Resource.Colors.bodyLight)
+                    }
+                    .onTapGesture {
+                        selectedMapMode = .offline
+                    }
+                }
+
+                Spacer()
+
+                Button {
+                    // Open the navigation app
```

Comment:
- Shall we make `showSheet` more descriptive so there is less need to read the comment? For example, `showNavigationExamplesSheet`
- I think we

Reply:
It's updated, and the comment has been removed.